libr/debug/session.c

```diff
@@ -244,9 +244,24 @@ R_API void r_debug_session_save(RDebug *dbg, const char *file) {
 	RSnapEntry snapentry;
 
 	ut32 i;
+	char *base_file, *diff_file;
+	const char *path = r_config_get (config, "dir.dbgsnap");
```

Comment:
this path should be cached in dbg->snappath;, you cant use rconfig inside rdebug